test/ui-test/src/screens/SettingsScreen.py

```diff
@@ -343,5 +351,25 @@ def change_user_password(self, oldPassword: str, newPassword: str):
 
         click_obj_by_name(ChangePasswordMenu.CHANGE_PASSWORD_SUBMIT_BUTTON.value)
         click_obj_by_name(ChangePasswordMenu.CHANGE_PASSWORD_SUCCESS_MENU_SIGN_OUT_QUIT_BUTTON.value)
-
+        
+    def add_contact_by_chat_key(self, chat_key: str, who_you_are: str):
+        click_obj_by_name(MessagingOptionScreen.CONTACTS_BTN.value)
+        click_obj_by_name(ContactsViewScreen.CONTACT_REQUEST_CHAT_KEY_BTN.value)
+        
+        type(ContactsViewScreen.CONTACT_REQUEST_CHAT_KEY_INPUT.value, chat_key)
+        type(ContactsViewScreen.CONTACT_REQUEST_SAY_WHO_YOU_ARE_INPUT.value, who_you_are)
+        
+        click_obj_by_name(ContactsViewScreen.CONTACT_REQUEST_SEND_BUTTON.value)
+
+    def verify_contact_request(self, chat_key: str):
+        time.sleep(0.5)
+        click_obj_by_name(ContactsViewScreen.CONTACT_REQUEST_PENDING_REQUEST_TAB_BUTTON.value)
+        contact_list = get_obj(ContactsViewScreen.SENT_REQUESTS_CONTACT_PANEL_LIST_VIEW.value)
+        for index in range(contact_list.count):
+            contact = contact_list.itemAtIndex(index)
+            print("Found: ", contact.compressedPk, ", Wanted:  ", chat_key)
```

Comment:
Is this a leftover or do we want to have "logging" into the std output?

Reply:
I wanted to leave it, since the verify failure at the bottom doesn't tell us much info if it fails

Reply:
Relying on std out for general logging generates noise after more and more output is added to it. I'm thinking that Maybe a more verbose verify will help improve the fail message and don't add up to become noisy output; e.g. `verify_failure(f'The found contact chat key "{contact.compressedPk}" doesn't match wanted chat key "{chat_key}" in the list of pending contacts')`